projectDocs/dev/developerGuide/developerGuide.t2t

```diff
@@ -1031,6 +1031,11 @@ For examples of how to define and use new extension points, please see the code
 || Type | Extension Point | Description |
 | ``Decider`` | ``decide_executeGesture`` | Notifies when a gesture is about to be executed, allowing other code to decide if it should be. |
 
+++ logHandler ++[logHandlerExtPts]
+
+|| Type | Extension Point | Description |
+| ``Action`` | ``getOnErrorSoundRequested()`` | Triggered every time an error sound needs to be played. |
```

Comment:
You are not documenting an extension point, but a function which returns, or creates and returns, an extension point. This is not how any other extension point is documented or used.
Either in the table description, or in a note immediately after the table, or both, this should be explained further.
A starting point may be:

```suggestion
| ``Action`` | ``getOnErrorSoundRequested()`` | Called to obtain the Action needed to notify when an error sound should be played. |

In order to avoid difficulties with early or circular importing, this extension point must be encapsulated by a function, and is silently created upon first use.
Other than calling it as a function, it may be used like any other extension point.
```

I admit that my understanding of your work in that module is vague at best, so I may have misunderstood something about how it works. But that's what I got from reading the code a minute ago, and what I remember from the original PR. Hopefully that helps some.